app/payment/channel_test.go

```diff
@@ -163,25 +169,26 @@ func Test_RespondPayChUpdate(t *testing.T) {
 	t.Run("happy_accept", func(t *testing.T) {
 		accept := true
 		chAPI := &mocks.ChAPI{}
-		chAPI.On("RespondChUpdate", context.Background(), updateID, accept).Return(nil)
+		chAPI.On("RespondChUpdate", context.Background(), updateID, accept).Return(updatedChInfo, nil)
 
-		gotErr := payment.RespondPayChUpdate(context.Background(), chAPI, updateID, accept)
+		gotPayChInfo, gotErr := payment.RespondPayChUpdate(context.Background(), chAPI, updateID, accept)
```

Comment:
Here, since `RespondPayChUpdate` is giving `updatedPayChInfo`, may be we can use the term like `updatedPayChInfo` so that we can understand from the name itself.

Reply:
Updated in [this commit](https://github.com/hyperledger-labs/perun-node/pull/116/commits/2f759e50555fbcb313b10428ddd4ab7ff753604a).